custom_components/tuya_local/devices/doel_tiplus_litterbox.yaml

```diff
@@ -55,37 +59,51 @@ secondary_entities:
       - id: 22
         type: bitfield
         name: fault_code
+      - id: 120
+        type: boolean
+        name: DP 120
```

Comment:
dp name should not have spaces, and by convention attribute names are all lower case.  I suggest unknown_120, though for 127 and 135, you did have a name you can use here, even if the function is not understood.